src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -144,12 +145,15 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         const { requestId } = meta || {};
         const { currentUser } = this.state;
         const isPending = action === 'create_start';
+        const currentFileVersionId = getProp(file, 'file_version.id');
+        // Patch the pending annotation with the current file version id
+        const patchedAnnotation = { ...annotation, file_version: { id: currentFileVersionId } };
```

Comment:
Can users create annotations on older versions?

Reply:
Good callout, patching with the current version won't work. I'll make the change upstream in box-annotations